support/tarball/prep.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
```

Comment:
This file has a permission of `0644`. It should be `0755` (executable by everyone), right?

Reply:
Thanks.  Fixed.